Formula/mariadb.rb

```diff
@@ -151,6 +151,9 @@ def caveats; <<-EOS.undent
 
     To connect:
         mysql -uroot
+
+    Using brew services will place a plist file at #{ENV["HOME"]}/Library/LaunchAgents/#{plist_name}
```

Comment:
Problem is if you change the plist it'll be overwritten by `brew services` in future. I think the `my.cnf` approach is probably the best bet.